### ISSUE TITLE
Feature request: Allow executing other process

### ISSUE THREAD

**Issue by christophgysin** [electrickite/caffeinated#1]
I would like to be able to start a process, and prevent the system from going to sleep until the process exits.
```
$ caffeinated [command...]
```
Incidentially, that would also provide an easy way to set a timeout, using e.g.
```
$ caffeinated sleep 1h
```

**Comment by electrickite** [comment]
On systems running systemd, you can use the `systemd-inhibit` to accomplish this. Since it already provides this capability, I'd rather not complicate caffeinated.

**Comment by christophgysin** [comment]
Fair enough. But if you assume everyone has systemd, what is it that this project provides that isn't already part of systemd?
